store/ssmstore_test.go

```diff
@@ -262,9 +262,8 @@ func matchStringFilters(filters []types.ParameterStringFilter, param mockParamet
 	return true, nil
 }
 
-func NewTestSSMStore(parameters map[string]mockParameter, usePaths bool) *SSMStore {
```

Comment:
I love how this test had a `usePaths` boolean argument but the actual code used a `noPaths` boolean argument. Twisted my brain right up!

Reply:
Oh geez, I don't know if I even noticed that